src/gluonnlp/models/bert.py

```diff
@@ -626,39 +623,33 @@ def layout(self):
     def forward(self, inputs, token_types, valid_length,
                        masked_positions):
         """Getting the scores of the masked positions.
-
```

Comment:
This line is required.

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line. The summary line may be on the same line as the opening quotes or on the next line. The entire docstring is indented the same as the quotes at its first line (see example below).

https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings